packages/expo-system-ui/ios/EXSystemUI/EXSystemUIModule.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021-present 650 Industries. (AKA Expo) All rights reserved.
+
+import ExpoModulesCore
+
+public class SystemUIModule: Module {
+  public func definition() -> ModuleDefinition {
+    name("ExpoSystemUI")
+
+    method("getBackgroundColorAsync") { () -> String? in
+      var color: String? = nil
+      EXUtilities.performSynchronously {
+        if let backgroundColor = self.appContext?.utilities?.currentViewController()?.view.backgroundColor?.cgColor {
+          color = EXUtilities.hexString(with: backgroundColor)
+        }
+      }
+      return color
+    }
+
+    method("setBackgroundColorAsync") { (color: Int) in
+      EXUtilities.performSynchronously {
+        self.appContext?.utilities?.currentViewController()?.view.backgroundColor = EXUtilities.uiColor(color)
+      }
+    }
+  }
+}
```

Comment:
The filename doesn't have to be prefixed with `EX` -> we don't prefix Swift files during versioning, but the module name instead 😉 
Also, tbh I'd get rid of additional `EXSystemUI` folder and just put everything in the `ios` folder (see the modules core, it needs glob path changes in the podspec as well). We can also think about renaming `EX*` pod names to `Expo*` (like `ExpoModulesCore`) because it's more clear where it comes from 🤔 What do you think?

Reply:
I'm not really sure, I guess the `EX` prefix is just from the legacy iOS standard which probably predates all of the package managers, in which case we probably don't need it here.